<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Значение</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insertButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти элемент</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ключ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>510</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить количество листов дерева</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>40</y>
      <width>331</width>
      <height>501</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="keyInsert">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000000000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="printButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="valueInsert">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>281</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>291</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать метод для вывода элементов</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>209</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="status">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="print">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="getCount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
